Makefile

```diff
@@ -0,0 +1,109 @@
+#! /usr/bin/env make -f
```

Comment:
Could you please add documentation to the `Makefile`?

Reply:
Do we want to add comments into the Makefile, or document it in more detail in our README?

Reply:
> Do we want to add comments into the Makefile, or document it in more detail in our README?

I think both?

Reply:
Commits https://github.com/jenstroeger/python-package-template/pull/74/commits/aa69524feffc597dc269fbb1d24c31902e7856f1 and https://github.com/jenstroeger/python-package-template/pull/74/commits/579f44cc3716fadd850d2ba10f9b12156275a647.